distribution/lib/Standard/Base/0.0.0-dev/src/System/File/Generic/File_Write_Strategy.enso

```diff
@@ -137,7 +138,8 @@ dry_run_behavior file behavior:Existing_File_Behavior -> Dry_Run_File_Settings =
    Generic `copy` implementation between two backends.
    The files only need to support `with_input_stream` and `with_output_stream`.
 generic_copy source destination replace_existing =
-    source.with_input_stream [File_Access.Read] input_stream->
-        options = if replace_existing then [File_Access.Write, File_Access.Create, File_Access.Truncate_Existing] else [File_Access.Write, File_Access.Create_New]
+    source.with_input_stream [File_Access.Read, Data_Link_Access.No_Follow] input_stream->
```

Comment:
Does this mean that it's impossible to read the target of a data link and copy its contents to another destination? Or is that possible elsewhere?

Reply:
One can always `read` and then `write` it, e.g. as bytes.

Just the `my_link.copy_to destination` copies the link and not the actual data, much like symlinks.